jujugui/static/gui/src/app/components/expert-card/experts.js

```diff
@@ -21,5 +21,22 @@ module.exports = {
     storeDescription: (
       'Learn how Spicule and Canonical can help solve your Big Data ' +
       'challenges with JAAS:')
+  },
+  'tengu-team': {
+    logo: 'tengu.png',
+    highlights: [
+      'Your own Big Data workspace',
+      'Use all Juju-charmed technologies',
```

Comment:
is this supposed to be "Uses"?

Reply:
This was taken from the copy that was supplied, I imagine it'll get an overhaul once it's on staging...